src/services/auth/sign-in/SignInPage.vue

```diff
@@ -102,6 +102,7 @@ export default {
         const state = reactive({
             userType: computed(() => (props.admin ? 'DOMAIN_OWNER' : 'USER')),
             authType: computed(() => store.state.domain.extendedAuthType),
+            beforeUser: store.state.user.userId,
```

Comment:
I think it would be better with `computed`
```suggestion
            beforeUser: computed<string>(() => store.state.user.userId),
```

Reply:
If this code use "calculated", it cannot compare previous userId.

Reply:
Oh, I got it